mock/middlewareMock.go

```diff
@@ -0,0 +1,47 @@
+package mock
```

Comment:
In terms of the file name, it's generally the convention to use snake_case for filenames. Furthermore it's unnecessary to repeat yourself in the package + filename.. I.E `mock/middleware.go` should suffice

Reply:
Changed.